libraries/chain/include/eosio/chain/webassembly/interface.hpp

```diff
@@ -1808,41 +1819,42 @@ namespace webassembly {
          int32_t bls_g2_add(span<const char> op1, span<const char> op2, span<char> result) const;
 
          /**
-          * Host function for G1 scalar multiplication on the elliptic curve bls12-381
+          * Host function for G2 addition on the elliptic curve bls12-381
           *
           * @ingroup crypto
-          * @param point - a span containing the G1 point operand.
-          * @param scalar - a span containing the scalar operand.
-          * @param[out] result - the result: scalar * point.
+          * @param op1 - a span containing the first operand G2 point in Montgemory form.
+          * @param op2 - a span containing the second operand G2 point in Montgemory form.
+          * @param[out] result - the result op1 + op2 in Montgemory form.
```

Comment:
```suggestion
          * @param op1 - a span containing the first operand G2 point in Montgomery form.
          * @param op2 - a span containing the second operand G2 point in Montgomery form.
          * @param[out] result - the result op1 + op2 in Montgomery form.
```

This typo occurs in many other places as well, please fix all.

Reply:
fixed